makiflow/layers/untrainable_layers.py

```diff
@@ -38,32 +40,38 @@ def __init__(self, input_shape, name):
         """
         self._name = str(name)
         self._input_shape = input_shape
+
         self._input = tf.placeholder(tf.float32, shape=input_shape, name=self._name)
+
         super().__init__(
             data_tensor=self._input,
             name=name
         )
 
     def __call__(self, x):
-        raise RuntimeError('This functionality is not implemented in the InputLayer.')
+        raise RuntimeError(InputLayer._ERROR_STRING_IS_NOT_IMPLEMENTED)
```

Comment:
An error is thrown regarding hardware issues such as lack of memory. In the case of 'code issue', an exception is thrown. Better change it from `_ERROR_STRING_IS_NOT_IMPLEMENTED` to `EXC_IS_NOT_IMPLEMENTED` or `EXCEPTION_IS_NOT_IMPLEMENTED`. The word 'string' is redundant.